OrbitCore/EventBuffer.cpp

```diff
@@ -12,12 +12,15 @@ using orbit_client_protos::CallstackEvent;
 
 EventTracer GEventTracer;
 
-std::vector<CallstackEvent> EventBuffer::GetCallstackEvents(
+std::vector<CallstackEvent> EventBuffer::GetCallstackEventsExcludingAllThreadsTid(
```

Comment:
nit. I would rename to GetCallstackEventsInTimeRangeForThreadId. "ExcludingAllThreadsTid" seems more confusing than anything.

Reply:
Done.